c7n/resources/rds.py

```diff
@@ -1426,6 +1426,10 @@ class RDSSnapshotDelete(BaseAction):
     permissions = ('rds:DeleteDBSnapshot',)
 
     def process(self, snapshots):
+        skipped = len([s for s in snapshots if s['SnapshotType'] == 'automated'])
```

Comment:
you can just use filter_resources which will also log from c7n/element.py

Reply:
Yeah we had a pass with that, it looked cleaner from the code side but less explicit on the output side. Fair point though, we can put it back to avoid duplicating that logic.